Server/JokesOnYou.Web.Api/Extensions/ServiceExtensions.cs

```diff
@@ -15,8 +19,20 @@ namespace JokesOnYou.Web.Api.Extensions
 {
     public static class ServiceExtensions
     {
-        public static IServiceCollection ConfigureAppServices(this IServiceCollection services, IConfiguration config)
+        public static void ConfigureAppServices(this IServiceCollection services, IConfiguration config)
         {
+            services.AddScoped<IUserService, UserService>();
+            services.AddScoped<ITokenService, JwtTokenService>();
+            services.AddScoped<ITagService, TagService>();
+
+            services.AddScoped<IUserRepository, UserRepository>();
+            services.AddScoped<IUnitOfWork, UnitOfWork>();
+            services.AddScoped<IJokesRepository, JokesRepository>();
+            services.AddScoped<IJokesService, JokesService>();
+            services.AddScoped<ITagRepository, TagRepository>();
```

Comment:
Maybe a bit overboard but.. should we give the scopes their own method?

Reply:
We can place all data related services in their own method